tqdm/_tqdm.py

```diff
@@ -101,7 +101,6 @@ def tqdm(iterable, desc=None, total=None, leave=False, file=sys.stderr,
         are displayed.
     file  : `io.TextIOWrapper` or `io.StringIO`, optional
         Specifies where to output the progress messages.
-        Uses file.write(str) and file.flush() methods.
```

Comment:
I added it because I thought it would be helpful... If you remove it that's more of a revert than a rebase.